### PR TITLE
#1624 fix player compare when sold player is bought back again

### PR DESCRIPTION
**Pull request by wsbrenk**
1. changes proposed in this pull request:
 
   - fixes issue #1624
   
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)
 
 
2. changelog and release_notes ...

 - [x] have been updated
 - [ ] do not require update

3. [Optional] suggested person to review this PR @tychobrailleur

### CODE REVIEW

**Comment by wsbrenk** [comment]
@tychobrailleur would you like to have a look over it here

**Comment by tychobrailleur** [comment]
I have gone to watch the final — will do depending on the outcome 😅

**Comment by wsbrenk** [comment]
@tychobrailleur final is over - you should have recovered... ;-)

**Comment by tychobrailleur** [comment]
> @tychobrailleur final is over - you should have recovered... ;-)

You would think... ;-)
I will look into it after the holidays!

File: src/main/java/module/playerOverview/SpielerTrainingsVergleichsPanel.java [hunk 1]
```diff
@@ -52,32 +55,36 @@ public SpielerTrainingsVergleichsPanel() {
...
 
-        for (int i = 0; (i < hrfs.length) && (i < 11); i++) {
-            text.append("\n").append(hrfs[i].toString());
-
-            if (i == 10) {
-                text.append("\n ... ");
+        int i=0;
+        for (var hrf : hrfs){
+            text.append("\n");
+            if ( i++ < 11){
```
Comment (tychobrailleur) [inline src/main/java/module/playerOverview/SpielerTrainingsVergleichsPanel.java:82]: Could you please extract that magic number?

**Comment by tychobrailleur** [comment]
This looks ok to me!

File: src/main/java/module/playerOverview/SpielerTrainingsVergleichsPanel.java (hunk 1 above)
Comment (wsbrenk) [reply src/main/java/module/playerOverview/SpielerTrainingsVergleichsPanel.java]: changed this - thanks

**Pull request merged**